docs/UPGRADES.md

```diff
@@ -1,8 +1,8 @@
 # CAPVCD Upgrades
 
-## Upgrades from CAPVCD 0.5.0 to 1.0.0
```

Comment:
We shouldn't overwrite older upgrade documentation, rather add new upgrade paths to the doc